packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -414,12 +396,21 @@ const TabHeader = React.forwardRef((props, forwardedRef) => {
       className={cx('iui-tab', { 'iui-active': isActive }, className)}
       role='tab'
       tabIndex={isActive ? 0 : -1}
-      onClick={mergeEventHandlers(onClickProp, () => onActiveChange?.())}
-      onKeyDown={onKeyDown}
       aria-selected={isActive}
       aria-controls={value}
       ref={useMergedRefs(tabRef, forwardedRef, setInitialActiveRef)}
       {...rest}
+      onClick={mergeEventHandlers(props.onClick, () => {
+        if (!props.disabled) {
```

Comment:
i think this check shouldn't be needed, as `ButtonBase` already checks for `disabled` in `onClick`

Reply:
It did not. I'll try again and do screen recording if failing.

Reply:
It selects tab with panel even when disabled without those checks


https://github.com/iTwin/iTwinUI/assets/83585998/c387ebf7-be4b-4745-a699-410a477a7b08

Reply:
are you sure? i can't reproduce.

note that i'm only talking about `onClick` (which affects `focusActivationMode='manual'`). it is fine to keep in `onFocus`.

![](https://github.com/iTwin/iTwinUI/assets/9084735/436e6abb-c2ee-43fd-876c-0fee15e74ff7)

Reply:
Yup, you are right, removed condition.